test/rng_testsuite/extra_methods_tests/engines_methods.pass.cpp

```diff
@@ -0,0 +1,263 @@
+// -*- C++ -*-
+//===-- engines_methods.hpp ----------------------             -------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+//
+// Abstract:
+//
+// Testing of different engines' methods
+
+#include "support/utils.h"
+#include <iostream>
+
+#if _ONEDPL_BACKEND_SYCL
+#    include <vector>
+#    include <CL/sycl.hpp>
+#    include <oneapi/dpl/random>
+
+#    define SEED 777
+#    define N_GEN 960
+#    define MINSTD_A 48271
+#    define MINSTD_C 0
+#    define MINSTD_M 2147483647
+#    define MINSTD_MIN 1
+#    define MINSTD_MAX 2147483646
+#    define RANLUX24_BASE_W 24
+#    define RANLUX24_BASE_S 10
+#    define RANLUX24_BASE_R 24
+#    define RANLUX24_BASE_MIN 0
+#    define RANLUX24_BASE_MAX 16777215
+#    define RANLUX24_P 223
+#    define RANLUX24_R 23

```

Comment:
Won't `constexpr` variables be better than macros here?

Reply:
changed to constants